specification.md

```diff
@@ -60,7 +60,15 @@
       - [FunctionRef Definition](#functionref-definition)
       - [EventRef Definition](#eventref-definition)
       - [SubFlowRef Definition](#subflowref-definition)
+      - [Error Handling Configuration](#error-handling-configuration)
       - [Error Definition](#error-definition)
+      - [Error Types](#error-types)
+      - [Error Reference](#error-reference)
+      - [Error Handler Definition](#error-handler-definition)
+      - [Error Handler Reference](#error-handler-reference)
+      - [Error Policy Definition](#error-policy-definition)
+      - [Error Outcome Definition](#error-outcome-definition)
+      - [Error Throw Definition](#error-throw-definition)
       - [Retry Definition](#retry-definition)
```

Comment:
Sure, great idea, will do!

Reply:
Actually, as I was going to fix it, I realized that it make for `AuthDefinition` to be structured in such a way, because its subsections are sub-concepts, which is not the case for `ErrorDefinition`.